src/main/java/shortestpath/transport/TransportLoader.java

```diff
@@ -0,0 +1,123 @@
+package shortestpath.transport;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Scanner;
+import java.util.Set;
+import lombok.extern.slf4j.Slf4j;
+import shortestpath.ShortestPathPlugin;
+import shortestpath.Util;
+import shortestpath.WorldPointUtil;
+
+@Slf4j
+public class TransportLoader {
+    private static final String DELIM_COLUMN = "\t";
+    private static final String PREFIX_COMMENT = "#";
+
+    private static void addTransports(Map<Integer, Set<Transport>> transports, String path, TransportType transportType) {
+        addTransports(transports, path, transportType, 0);
+    }
+
+    private static void addTransports(Map<Integer, Set<Transport>> transports, String path, TransportType transportType, int radiusThreshold) {
+        try {
+            String s = new String(Util.readAllBytes(ShortestPathPlugin.class.getResourceAsStream(path)), StandardCharsets.UTF_8);
+            addTransportsFromContents(transports, s, transportType, radiusThreshold);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public static void addTransportsFromContents(Map<Integer, Set<Transport>> transports, String contents, TransportType transportType, int radiusThreshold) {
+        Scanner scanner = new Scanner(contents);
+
+        String headerLine = scanner.nextLine();
+        headerLine = headerLine.startsWith(PREFIX_COMMENT + " ") ? headerLine.replace(PREFIX_COMMENT + " ", PREFIX_COMMENT) : headerLine;
+        headerLine = headerLine.startsWith(PREFIX_COMMENT) ? headerLine.replace(PREFIX_COMMENT, "") : headerLine;
+        String[] headers = headerLine.split(DELIM_COLUMN);
+
+        Set<Transport> newTransports = new HashSet<>();
+
+        while (scanner.hasNextLine()) {
+            String line = scanner.nextLine();
+
+            if (line.startsWith(PREFIX_COMMENT) || line.isBlank()) {
+                continue;
+            }
+
+            String[] fields = line.split(DELIM_COLUMN);
+            Map<String, String> fieldMap = new HashMap<>();
+            for (int i = 0; i < headers.length; i++) {
+                if (i < fields.length) {
+                    fieldMap.put(headers[i], fields[i]);
+                }
+            }
+
+            Transport transport = new Transport(fieldMap, transportType);
+            newTransports.add(transport);
+
+        }
+        scanner.close();
+
+        Set<Transport> transportOrigins = new HashSet<>();
```

Comment:
The extensive comment I had here seems to be gone. So are other comments also it seems. Is this intentional?

Reply:
good catch, that was not intentional, and now it's back